spec/helpers/brexit_checker_helper_spec.rb

```diff
@@ -199,4 +199,58 @@
       expect(link).to match("http://www.gov.uk?")
     end
   end
+
+  describe "#email_link_label" do
+    let(:actions) { [FactoryBot.build(:brexit_checker_action)] }
+
+    it "Should return the email link copy if there are actions" do
+      expect(email_link_label(actions)).to eq(t("brexit_checker.results.email_sign_up_link"))
+    end
+
+    it "Should return the no results email link copy if there are no actions" do
+      expect(email_link_label([])).to eq(t("brexit_checker.results.email_sign_up_link_no_actions"))
+    end
+  end
+
+  describe "#title" do
+    let(:actions) { [FactoryBot.build(:brexit_checker_action)] }
+    let(:criteria_keys) { %w"nationality-eu" }
+
+    it "Should return the title if there are actions and answers" do
```

Comment:
Minor: the word "should" implies some level of doubt 😆. Could we just say `it "returns..."`.

Reply:
fair cop!
Amended